src/panels/config/entities/dialog-entity-editor.ts

```diff
@@ -144,15 +197,33 @@ export class DialogEntityRegistryDetail extends LitElement {
     fireEvent(this._dialog as HTMLElement, "iron-resize");
   }
 
+  private async _loadPlatformSettingTabs(): Promise<void> {
+    if (!this._entry) {
+      return;
+    }
+    if (PLATFORMS_WITH_SETTINGS_TAB.includes(this._entry.platform)) {
+      await import(`./editor-tabs/entity-platform-tab-${this._entry.platform}`);
```

Comment:
When you write a dynamic import statement, Webpack will try to figure out all the different values and pack those up. So this is good because we have a prefix, but be careful :) I usually prefer explicit imports (like we do for lazy load types in create card element)